android/KMEA/app/src/main/assets/other-polyfills.js

```diff
@@ -0,0 +1,17 @@
+// Needed by the KMW's gesture engine until Chrome 61.
+var DOMRect = DOMRect || function (x, y, width, height) {
+  this.x = this.left = x;
+  this.y = this.top = y;
+  this.width = width;
+  this.height = height;
+  this.bottom = y + height;
+  this.right = x + width;
+};
+
+// https://stackoverflow.com/questions/53308396/how-to-polyfill-array-prototype-includes-for-ie8
+if(!Array.prototype.includes){
+  //or use Object.defineProperty
+  Array.prototype.includes = function(search){
+   return !!~this.indexOf(search);
+ }
+}
```

Comment:
1. Is there any syntax required for this polyfill to work with KeymanWeb's ES Modules?

2.
https://github.com/keymanapp/keyman/blob/33eae05cb0fa24e87a6193a8ae0a56060179f873/android/KMEA/build.sh#L95-L96

If this is the only KeymanWeb "asset" not copied from KeymanWeb, I'd probably add a comment about "other-polyfills.js" not needing to be copied.

Reply:
`other-polyfills.js` is hand-maintained, not being copied from a Node package.  The only "copying" done here was copying the `Array.prototype.includes` polyfill used by the lm-worker into other-polyfills.js.